js/src/wp-seo-featured-image.js

```diff
@@ -116,6 +116,17 @@ import { isGutenbergDataAvailable } from "./helpers/isGutenbergAvailable";
 		}
 	}
 
+	/**
+	 * Checks whether the featuredImageId is invalid.
+	 *
+	 * @param {any} featuredImageId The possibly invalid featuredImage Id.
+	 *
+	 * @returns {boolean} True if the featured image id is valid.
```

Comment:
As a general consideration, sounds a bit weird to me checking if something is _not_ valid. I'd rather check for the opposite case. It would also be simpler: a featured image ID is valid when it's a number and it's greater than `0`. No need to check for undefined or null.

Reply:
The reason I did it like this is that if I would check for valid, I would need to add a `!` in this function, and add a `!` in the if where this function was called. This reversion in two places didn't seem optimal to me.  I think the "number and greater than 0" check is a nice solution. Thanks!